File: Assignment 12/Question1.c
#include <stdio.h>

void FactorDisplay(int iNo)
{
    int iCnt = 0 ;
    for(iCnt = 1 ; iCnt<=(iNo/2) ; iCnt++)
    {
        if(iNo % iCnt ==0)
        {
            printf("%d\t",iCnt);
        }
    }
}

int main()
{
    int iValue = 0 ;


    printf("Enter Number : ");
    scanf("%d",&iValue);

    FactorDisplay(iValue);

    return 0;
}

File: Assignment 16/Question1.java
import java.util.Scanner;
class Logic
{
    public int iSum = 0;
    public int iCnt = 0 ;

    public void Sum(int iNo)
    {
        for(iCnt = 1 ; iCnt<=iNo ; iCnt++)
        {
            iSum = iSum + iCnt ;
            
        }
        System.out.println(iSum);
    }
}

class Question1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;

        System.out.println("Enter Your Number");
        iValue = sobj.nextInt();

        Logic lObj = new Logic();

        lObj.Sum(iValue);
        




    }
}